src/main/webapp/index.html

```diff
@@ -18,75 +18,55 @@
     <p>Rate your classes and professors!</p>
     <form action="/data" method="POST">
       <p>Rate your class:</p>
-      <div id="class-id"></div>
-      <button onclick="getClassID()">Get class ID</button>
       <br /><br />
 
-      <textarea name="class-input">Write opinion on class here.</textarea>
+      <textarea name="class-input" maxlength="1500">
+Write opinion on class here.</textarea
+      >
```

Comment:
Was this a prettier formatting this? I don't get the > on one line by itself.

Reply:
Yeah, prettier formatting but no clue what that is doing.